contract/src/pegasus.js

```diff
@@ -306,26 +231,14 @@ const makePegasus = (zcf, board) => {
    */
   const connectionToLocalDenomState = makeWeakStore('Connection');
 
-  /**
-   * @type {WeakStore<Brand, Issuer>}
-   */
-  const brandToIssuer = makeWeakStore('Brand');
-
   let lastLocalIssuerNonce = 0;
   /**
-   * Register a brand and issuer with the zcf.
-   * @param {Brand} localBrand
-   * @param {Issuer} localIssuer
+   * Create a new issuer keyword (based on Local + nonce)
+   * @returns {string}
    */
-  const registerBrand = async (localBrand, localIssuer) => {
-    if (brandToIssuer.has(localBrand)) {
-      // Already exists.
-      return;
-    }
+  const createLocalIssuerKeyword = () => {
     lastLocalIssuerNonce += 1;
-    const localIssuerKeyword = `Local${lastLocalIssuerNonce}`;
-    await zcf.addNewIssuer(localIssuer, localIssuerKeyword);
-    brandToIssuer.init(localBrand, localIssuer);
+    return `Local${lastLocalIssuerNonce}`;
   };
 
   /**
```

Comment:
Line 273 below
```js
    updater.updateState([...pegToConnection.keys()]);
```
is a pattern we use several places, to notify of incremental changes to a collection by sending the whole collection, which is O(N**2) on the sender's work, on the packets, and on the receiver's work. In this case N is tiny, so it is likely fine for now. Possibly for all other current cases too. Just noting that a lossless incremental notifier would be nice to have. No change suggested for this PR.

Reply:
> Just noting that a lossless incremental notifier would be nice to have

Agreed.  Preferably some combination of "absolute" updates that throw away all prior state, and "relative" updates that contain differences since the last "absolute" state.  That would be useful in UIs, too.